broker/src/main/java/io/camunda/zeebe/broker/partitioning/PartitionManagerImpl.java

```diff
@@ -182,7 +183,14 @@ private void completePartitionStart(
       final int partitionId, final Throwable error, final ActorFuture<Void> future) {
 
     if (error != null) {
-      LOGGER.error("Failed to start partition {}", partitionId, error);
+      // If Partition start was not complete due to a shutdown being called
+      // during the startup process, then this shouldn't be logged as an error
+      if (error.getClass() == StartupProcessShutdownException.class) {
+        LOGGER.warn("Failed to start partition {}", partitionId, error);
```

Comment:
:wrench: I think debug log is enough. 
```suggestion
        LOGGER.debug("Aborting startup of partition {}", partitionId, error);
```